storage/replica.go

```diff
@@ -1987,6 +1987,24 @@ func (r *Replica) tickRaftMuLocked() (bool, error) {
 		return false, nil
 	}
 	if r.mu.quiescent {
+		// While a replica is quiesced we still advance its logical clock. This is
+		// necessary to avoid a scenario where the leader quiesces and a follower
+		// does not. The follower calls an election but the election fails because
+		// the leader and other follower believe that no time in the current term
+		// has passed. The Raft group is then in a state where one member has a
```

Comment:
Might wanna mention that this is an election thrashing prevention method toggled by the `CheckQuorum` option (enabled at time of writing).